<!-- original file: raspimouse_ros_2/launch/raspimouse.launch -->

<launch>
  <!-- for imu -->
  <!-- <arg name="imu" default="0" /> -->
  <!-- <include if="$(arg imu)" file="$(find rt_usb_9axis_sensor)/launch/rt_usb_9axis_sensor.launch" /> -->
  
  <!-- for real machine -->
  <!-- <include file="$(find pimouse_ros)/launch/pimouse.launch" /> -->
  <!-- <node pkg="raspimouse_ros_2" name="buzzer" type="buzzer.py" required="true" /> -->

  <!-- <node pkg="raspimouse_ros_2" name="lightsensors" type="lightsensors" required="true">
    <param name="frequency" value="10" />
  </node> -->

  <!-- <arg name="initial_motor_power" default="off" /> -->
  <!-- <node pkg="raspimouse_ros_2" name="motors" type="motors" required="true" output="screen" args="$(arg initial_motor_power)" /> -->

  <!-- <node pkg="raspimouse_ros_2" name="leds" type="leds" required="true" /> -->
  <!-- <node pkg="raspimouse_ros_2" name="buttons" type="buttons" required="true" /> -->

  <!-- for simulation -->
  <!-- <node pkg="raspimouse_sim_tutorial_program" name="LeftHand" type="left_hand.py" required="true" /> -->
  <!-- パラメータ設定（シミュ:0/実機:1） -->
  <param name="sim_act" value="0" />
  <node pkg="pimouse_sim_act" name="LeftHand" type="left_hand3.py" required="true" />
</launch>
